processor/processor.go

```diff
@@ -3013,20 +3139,20 @@ func (proc *Handle) transformSrcDest(
 				diffMetrics := getDiffMetrics(
 					reportingtypes.EVENT_FILTER,
 					reportingtypes.DEST_TRANSFORMER,
-					inCountMetadataMap,
-					successCountMetadataMap,
-					inCountMap,
+					data.inCountMetadataMap,
+					nil,
```

Comment:
was it a bug in the previous implementation that we were using `successCountMetadataMap`?

Reply:
I felt so.. It doesn't look like the successCountMetadataMap was generated from dest transform call but to maintain the old behaviour we can add it to the struct